apps/openmw/mwworld/cellstore.hpp

```diff
@@ -357,6 +362,12 @@ namespace MWWorld
         return mWeapons;
     }
 
+    template<>
+    inline CellRefList<ESM::Door>& CellStore::getReadOnly<ESM::Door>()
```

Comment:
Why not make this function return a const reference? Am I missing something here?